scanners/amass/parser/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "amass-parser",
+  "version": "1.0.0",
+  "description": "secureCodeBox parser for amass",
+  "main": "parser.js",
+  "scripts": {
+    "test": "jest"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/securecodebox/securecodebox.git"
+  },
+  "keywords": [
+    "amass",
+    "scanner",
+    "secureCodeBox"
+  ],
+  "author": "iteratec GmbH",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/securecodebox/securecodebox/issues"
+  },
+  "homepage": "https://github.com/securecodebox/securecodebox#readme",
+  "devDependencies": {
+    "@kubernetes/client-node": "^0.14.3",
```

Comment:
From a maintainer perspective it would be awesome if we could find a way to run the tests without having every scanner bring their own package.json & jest installation as this will cause dependabot to open up dozens of PR's on new vulnerabilities.

Maybe we could use `npx jest` / `npx -p jest@27 jest` from the makefiles? Would probably need to have kubernetes client-node installed either on the top level in the repo / globally to work.

Reply:
The Problem is that some parsers/hooks like finding-post-processing have their own dependencies.